### ISSUE TITLE
KYC from one bank is not view able by any other registered bank

### ISSUE THREAD

**Issue by AN8991** [rudrakh97/KYC-chain#5]
I have 2 bank accounts registered. Lets take them X & Y. So if X has done a KYC for customer A and A has approved the request then X can view the KYC Details accordingly. Now lets say that Y is interested in knowing the KYC Details of Customer A, then they need to accordingly search by username which then should trigger a pull request to the customer A for approval. This is the case where the application displays the message "Access denied! Take permission from the customer to proceed". However the request does not show up in the "View Request" tab of the Customer Profile. 

This leads to a gap where Bank Y is not able to ever view the KYC for Customer A. Has this been deliberately done in functionality or is it a miss? It would be great if you could clarify.

**Comment by suneelkumar91** [comment]
@AN8991  did you get any clarification about this flow ?

**Comment by AN8991** [comment]
Nope, not yet. There is yet another small bug present. If you do not click on "View KYC" button then the approval request doesnot flow to the customer. I am a little busy in other activities right now , but hopefully i should be making the required changes in the code.

**Comment by suneelkumar91** [comment]
@AN8991  Thanks for update !!! 
Yes I have notice this bug as well. 
I am look forward bug free code with complete flow.

**Comment by tussipandey** [comment]
@AN8991  @suneelkumar91  I am also facing the same problem.Did you find the solution of the mentioned bug  ??
